docs/bsp.md

```diff
@@ -1240,13 +1237,29 @@ trait SbtBuildTarget {
   /** The classpath for the sbt build (including sbt jars). */
   def classpath: List[Uri]
   
-  /** The Scala build target associated with this sbt build.
-    * It contains the scala version and the scala jars used. */
+  /** The Scala build target describing the scala
+   * version and scala jars used by this sbt version. */
   def scalaBuildTarget: ScalaBuildTarget
+  
+  /** An optional parent if the target has an sbt meta project. */
+  def parent: Option[BuildTargetIdentifier]
+  
+  /** The build targets defined by the sbt build represented
+    * in this target. It can contain normal project targets or sbt
+    * build targets if this target represents an sbt meta-meta build. */
+  def definedTargets: List[BuildTargetIdentifier]
```

Comment:
Should this be named `children`? Judging by the description it seems like the inverse of `parent`

Reply:
I agree with you this is more consistent with `parent`, I've just renamed it